docs/checks/internal/checks.yaml

```diff
@@ -531,10 +531,10 @@ checks:
       codebase.
 
       The checks currently looks for known Github apps such as
-      [CodeQL](https://codeql.github.com/) (github-code-scanning),
-      [LGTM](https://lgtm.com/) and
+      [CodeQL](https://codeql.github.com/) (github-code-scanning) or
       [SonarCloud](https://sonarcloud.io/) in the recent (~30) merged PRs, or the use
-      of "github/codeql-action" in a GitHub workflow.
+      of "github/codeql-action" in a GitHub workflow. The deprecated [LGTM](https://lgtm.com/)
+      service is also checked.
```

Comment:
Minor nit, just switch from passive to active ("It also checks for the deprecated LGTM service.")

What happens after December when the service is shut down? This could specify, "It will also check the deprecated LGTM service until it is shut down in December 2022.")

Reply:
did "forthcoming shutdown" to avoid any weirdness if dates move (that never happens!)

Reply:
perfect, thanks!